content/1-reactivity/1-declare-state/aurelia1/name.ts

```diff
@@ -0,0 +1,3 @@
+export class Name {
```

Comment:
How "name.html" is related to this file ?

Reply:
See last comment: by convention.